cuisino/templates/restaurant/menutest.html

```diff
@@ -0,0 +1,52 @@
+{% extends 'users/base.html' %}
```

Comment:
This file was used for me to test some stuff. There are no connections to this file.